shared/src/metabase/mbql/util.cljc

```diff
@@ -52,6 +52,12 @@
      ((set k-or-ks) (first x))
      (= k-or-ks (first x)))))
 
+(defn get-clause
```

Comment:
the name makes me a bit confused, if `x` is a clause, then `(rest x)` must be sth else right?

Reply:
Hmm... well, it feels like a good name to me, i.e. if you "get" some clause, you get the body of it, not the identifier. Could be `get-clause-body`, but that makes it less succinct.

Reply:
`get-clause-body` makes more sense to me. I think it's crucial to get the name for utils function, because most of the time we shouldn't look into its implementation to know what it does.

Reply:
and it'll be better if you write an example in the docstring

```
(get-clause [:field 1 nil] :field) => [1 nil])
```

Reply:
Haha, clearly you feel strongly about that one. But on the other hand `get` is not called `get-value`, while it gets the value, right?

I'd look at this as a part of lingo, in _any_ case, you'll have to understand what exactly the function does to your data. And given function similarity to `is-clause?`, I really like the name. :)

TBH now that I think about it, I really love the idea of something like this:

```
(defn check-clause [k-or-ks x]
  (when (is-clause? k-or-ks x) x))
```

What do you think?

Reply:
And then used like that:

```
(defn- get-template-tag
  "Fetch the `:field` clause from `dashcard` referenced by `:template-tag`.

    (get-template-tag [:template-tag :company] some-dashcard) ; -> [:field 100 nil]"
  [dimension card]
  (when-let [[_ tag] (mbql.u/check-clause :template-tag dimension)]
    (get-in card [:dataset_query :native :template-tags (u/qualified-name tag)])))
```